fecgov/fec-cms
# Add MUR date filter

## Summary (required)

- Resolves #2969 
_This adds new case open and case close date range filters to the MUR datatable search._

**TODO in future ticket:**
Page still refreshes every time a query is submitted or the page paginates. There are future enhancements we'll need to make in order to update this. Ticket has been created here: https://github.com/fecgov/fec-cms/issues/3037

## Impacted areas of the application
List general components of the application that this PR will affect:

- http://localhost:8000/data/legal/search/murs/

**Side note:** I needed to update the ADRs and Admin Fines datatable templates in order to get the filters indented correctly whenever we do add accordions into the page. This should not change too much visually for those tables except for slight padding up at the top, but still worth checking against feature.

For these pages, you'll need to turn on those feature flags: 
`export FEC_FEATURE_ADRS=true` and `export FEC_FEATURE_AFS=true`

- http://localhost:8000/data/legal/search/adrs/ and check against feature space https://fec-feature-cms.app.cloud.gov/data/legal/search/adrs/
- http://localhost:8000/data/legal/search/admin_fines/ and check against feature space https://fec-feature-cms.app.cloud.gov/data/legal/search/admin_fines/

## Screenshots

<img width="1680" alt="Screen Shot 2019-07-10 at 2 39 57 PM" src="https://user-images.githubusercontent.com/12799132/60995512-a2601e80-a320-11e9-897c-611c7a3bfd90.png">

## How to test
1. Check out this branch: feature/2969-mur-date-filter. These are pure python related changes, so you don't really need to do any builds. But it doesn't hurt to do it if you want to `npm run build`.
2. Go to http://localhost:8000/data/legal/search/murs/
3. Try performing some date range searching for both `CASE OPEN DATE RANGE` and `CASE CLOSE DATE RANGE`. Make sure the results are updating as expected.

____
